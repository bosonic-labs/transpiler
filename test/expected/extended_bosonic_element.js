(function () {
    var BModalPrototype = Object.create(BDialog.prototype, {
            dummy: {
                enumerable: true,
                value: function () {
                    console.log('bar');
                }
            }
        });
    window.BModal = document.registerElement('b-modal', { prototype: BModalPrototype });
    Object.defineProperty(BModal.prototype, '_super', {
        enumerable: false,
        writable: false,
        configurable: false,
        value: BDialog.prototype
    });
}());