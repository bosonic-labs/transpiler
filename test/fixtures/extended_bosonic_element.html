<element name="b-modal" extends="b-dialog">
  <script>
    ({
      dummy: function () {
        console.log('bar');
      }
    });
  </script>
</element>