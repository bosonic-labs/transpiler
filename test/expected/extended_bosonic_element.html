<element name="b-modal" extends="b-dialog">
  <script>
(function () {
    window.BModal = document.registerElement('b-modal', {
        prototype: Object.create(BDialog.prototype, {
            dummy: {
                enumerable: true,
                value: function () {
                    console.log('bar');
                }
            }
        }),
        extends: 'b-dialog'
    });
    Object.defineProperty(BModal.prototype, '_super', {
        enumerable: false,
        writable: false,
        configurable: false,
        value: BDialog.prototype
    });
}());
</script>
</element>